addons/mail/static/tests/message/message_reply.test.js

```diff
@@ -120,3 +121,28 @@ test("click on message in reply highlights original message", async () => {
     );
     await contains(".o-mail-Message.o-highlighted:contains('This message has been removed')");
 });
+
+test("reply action functionality in Chatter", async () => {
+    const pyEnv = await startServer();
+    const partnerBId = pyEnv["res.partner"].create({ name: "Partner B" });
+    pyEnv["mail.message"].create({
+        body: "Test message from B",
+        subtype_id: pyEnv["mail.message.subtype"].search([
+            ["subtype_xmlid", "=", "mail.mt_note"],
+        ])[0],
+        model: "res.partner",
+        res_id: serverState.partnerId,
+        author_id: partnerBId,
+    });
+    await start();
+    await openFormView("res.partner", serverState.partnerId);
+    await click(".o-mail-Message-actions button[title='Reply']");
```

Comment:
Would target `.o-mail-Message` (not `-actions`). The `-actions` is a detail, actually some residual classnames from when action template was all made by hand in message template. There's work in progress to rely fully on `ActionList` therefore the classname won't exist anymore, so could be nice to already avoid this occurrence if you can :)